save-cloud-charts/save-cloud/values-minikube.yaml

```diff
@@ -30,10 +30,14 @@ demo_cpg:
 demo:
   profile: dev
   dockerHost: tcp://${HOST_IP}:2376
+  applicationProperties: |
+    demo.kubernetes.useGvisor=false
 
 mysql:
   external: false
   ip: nil
+  migrations:
+    enabled: false
```

Comment:
The point is that `initContainers` work weird. It is better to set port-forwarding for `3306` and run `liquibaseUpdate` manually

Reply:
manual work instead of automatization?

googled a little bit, looks like k8s doesn't have a dependency between services\pods. I guess the problem with `liquibaseUpdate` that mysql is not running at this time.
But you can look here https://kubernetes.io/docs/concepts/workloads/pods/init-containers/ example with using `busybox:1.28` which waiting services

Reply:
Will add `busybox` pod that waits until database pod is ready (only in `external == false`) in next pr

Reply:
#1871